site/src/components/Hero.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+// assets import
+import cover from "../assets/images/hero/cover.svg";
+import graphic from "../assets/images/hero/graphic.svg";
+
+
+// styles import
+import HeroStyleWrapper from "../sections/hero.style";
+
+// ui import
+import { StaticImage } from "gatsby-plugin-image";
+
+const Hero = () => {
+  return (
+    <HeroStyleWrapper>
+      <img className="graphic" src={graphic} alt="Hero Section Graphic" />
+      <div className="content">
+        <div className="info">
+          <h1 className="title">Service Mesh Patterns</h1>
+          <h3 className="description">Best patterns and practices for getting the most out of Istio, Linkerd, Consul, App Mesh, or any service mesh, including those using Envoy and WebAssembly.</h3>
+          <div className="credit">
+            <h3>By</h3>
```

Comment:
Use `p`

Reply:
Both.
Everything except the `header` "Service Mesh Patterns` should be `p`

Reply:
okay thanks. I've applied all the review changes and here's a preview, please let me know what else I should change

![image](https://user-images.githubusercontent.com/96642972/183577031-71bd283f-5db6-4b37-9eff-6b3289eca2f0.png)

Reply:
Resolved